### PR TITLE
minor text changes

### PR DESCRIPTION
**Pull request by JoelLucaAdams**

### CODE REVIEW

**Review by IdrisTheDragon — CHANGES_REQUESTED** [review]
add the .idea folder to the .gitignore so we don't have loads of changes from it, tbf I should have done this already.

File: demobot.py [hunk 1]
```diff
@@ -44,15 +44,15 @@ async def on_ready():
 @bot.event
 async def on_command_error(ctx, error):
     if isinstance(error, commands.errors.CheckFailure):
-        await ctx.send('You do not have the correct role for this command.')
+        await ctx.send('You do not have the correct role for this command')
     elif isinstance(error, commands.errors.MissingRequiredArgument):
-        await ctx.send('You are missing a required argument.')
+        await ctx.send('You are missing a required argument')
```
Comment (IdrisTheDragon) [inline demobot.py]: you are removing a full stop here?

File: demobot.py [hunk 2]
```diff
@@ -68,43 +68,43 @@ async def add(ctx:Context):
     if s not in q:
         q.append(s)
         logging.info('{0} add {1}'.format(ctx.guild,s))
-        await ctx.send('added you to the queue,\n' + getCustomAddMessage(ctx.guild))
+        await ctx.send(s + ' has been added to the queue. ' + getCustomAddMessage(ctx.guild))
```
Comment (IdrisTheDragon) [inline demobot.py:71]: but adding one in here?

Comment (JoelLucaAdams) [reply demobot.py:71]: yeah added one here so the new custom message appears after the original message

File: demobot.py (hunk 1 above)
Comment (JoelLucaAdams) [reply demobot.py]: don't remember changing this at all... weird

**Review by IdrisTheDragon — APPROVED** [review]

**Pull request merged**